CMakeLists.txt

```diff
@@ -56,6 +56,9 @@ install(
 ament_export_include_directories(include)
 ament_export_libraries(${PROJECT_NAME})
 ament_export_targets(${PROJECT_NAME})
+ament_export_dependencies(rclcpp)
+ament_export_dependencies(rcutils)
+ament_export_dependencies(builtin_interfaces)
```

Comment:
Nit: this can all be done on one line:

```
ament_export_dependencies(rclcpp rcutils builtin_interfaces)
```

Reply:
Done in https://github.com/ros2/message_filters/pull/54/commits/935c87fa655b3476ffb072af28c5d144f4328a19.